server/docker-compose.yml

```diff
@@ -20,11 +20,6 @@ services:
 
   pgbouncer:
     image: edoburu/pgbouncer:1.15.0
-    healthcheck:
-      test: "pg_isready -h localhost"
-      interval: 30s
-      timeout: 10s
-      retries: 3
```

Comment:
Unrelated, please revert this change.

Reply:
Oops -- removed it from the wrong  `docker-compose`